front/src/components/Menu/index.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { useSelector } from "react-redux";
+import AppBar from "@material-ui/core/AppBar";
```

Comment:
Czysciej jak importujesz 
import {AppBar, Toolbar, Typography, ...} from "@material-ui/core";
nie ma to juz wplywu na performance - kiedys mialo i faktycznie lepiej bylo importowac po jednym komponencie

Reply:
super tip :) 
Done.